TestAssets/LockFileSamples/ExportFileSamples/valid/project.lock.json

```diff
@@ -0,0 +1,61 @@
+{
```

Comment:
Export files appear near a specialization of Lock files. They don't exist without lock files, and when they do exist, the lock files are special, in the sense that they have holes in them

I'll remove the samples bit.